tests/unit/amo/components/TestAddon.js

```diff
@@ -44,7 +44,10 @@ import {
   dispatchSignInActions, fakeAddon, signedInApiState,
 } from 'tests/unit/amo/helpers';
 import {
-  createFetchAddonResult, getFakeI18nInst, sampleUserAgentParsed,
+  createFetchAddonResult,
+  getFakeI18nInst,
+  sampleUserAgentParsed,
+  createStubErrorHandler,
```

Comment:
That's cool. I dislike having to set-up pre-commit hooks but I guess the bash script could just be part of the tests we expect devs to run. I'm certainly not opposed as long as it's frictionless, not sure if it is though.